app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -32,6 +31,7 @@
     @BindView(R.id.fragment_container)
     ViewPager viewPager;
     private SlidingTabAdapter tabAdapter;
+    private boolean nextVisible = true;
```

Comment:
would we have the fragment just return a boolean value that the view can request? It is a bit difficult to programmatically change the menu's configurations if one of these classes isn't retaining the boolean as a member variable. @jmccormack200